schemas/dex/SCHEMA.md

```diff
@@ -78,85 +96,92 @@ This is OpenBlock Labs standard DEX schema.
+| contract_address      | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address         | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
 | pool_address          | The contract address of the pool.                                    | string |
 | pool_token_balance    | The balance of the pool token, decimal normalized.                   | number |
 
 ### V3 Events
 | Property                | Description                                                      | Type   |
 |-------------------------|------------------------------------------------------------------|--------|
-| timestamp               | The timestamp of the record.                                     | number |
-| chain_id                | The standard id of the chain.                                    | number |
-| block_number            | The block number of the activity.                                | number |
+| timestamp               | The timestamp of the transaction.                                    | number |
+| chain_id                | The standard id of the chain.                                        | number |
+| block_number            | The block number of the trade.                                       | number |
+| log_index               | The log index of the event recorded.                                 | number |
+| transaction_hash        | The hash of the transaction.                                         | string |
+| user_address            | The address that initiates the transaction (ie, the transaction signer). | string |
+| contract_address        | The smart contract address the user interacted with (ie, transaction to). | string |
+| taker_address           | The taker, the address that receives the output of the swap (ie, could be the same as user_address unless a proxy contract/router is used). | string |
 | event_type              | The action taken (Mint/Burn/Transfer).                           | string |
-| user_address            | The address of the user who made the event.                      | string |
-| transaction_from_address| The address from which the transaction originated.               | string |
-| transaction_to_address  | The address to which the transaction is directed.                | string |
 | pool_address            | The contract address of the pool.                                | string |
 | token_address           | The contract address of the token.                               | string |
-| liquidity_amount        | The amount of liquidity involved.                                | number |
-| liquidity_usd           | The liquidity amount in USD.                                     | number |
+| liquidity_amount        | The amount of liquidity, decimal normalized.                                | number |
```

Comment:
there is no normalization for this, its the amount of liquidity minted to the position range, that is used for calculating token0 and token1. 

https://docs.uniswap.org/contracts/v3/reference/core/interfaces/pool/IUniswapV3PoolEvents